packages/aws-lambda-graphql/src/Server.ts

```diff
@@ -239,6 +239,10 @@ export class Server<
 
             return {
               body: '',
+              headers:
+                event.headers['Sec-WebSocket-Protocol'] === 'graphql-ws'
```

Comment:
Please change this to return the value of `Sec-WebSocket-Protocol` instead of constant.

Reply:
Updated.